### PR TITLE
[MRG] Automated Docker builds

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
This is ready for review/merging

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+             make ca-certificates python-pip python-wheel python-enchant
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ${PACKAGES} && \
+    apt-get clean
+
+WORKDIR /home
+
+ARG branch
+ENV branch ${branch:-v2.1.1}
```
Comment (standage) [inline docker/Dockerfile]: Just to clarify: the `${branch:-v2.1.1}` syntax defines a default value if otherwise undefined, right?

**Review by standage — APPROVED** [review]
Looks good. Thanks!

File: docker/Dockerfile (hunk 1 above)
Comment (mr-c) [reply docker/Dockerfile]: Yep!

**Comment by mr-c** [comment]
Thanks for the merge :-)

**Pull request merged**